### PR TITLE
[MRG] Add link to circleci artifacts

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
This is a test to see if we can generate a link to the artifacts in the PRs

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
This might need to be merged to master to be active (I'm not very good at github workflows, sorry).

@nilearn/core : unless someone objects, I will merge, to continue the experimentation on master.

**Comment by kchawla-pi** [comment]
We should first fix the failing tests by #2321 and merge that into master, make sure the tests here are passing, before merging this.

**Comment by GaelVaroquaux** [comment]
> We should first fix the failing tests by #2321 and merge that into master, make sure the tests here are passing, before merging this.

Maybe. On the other hand, this PR touches no code at all, so I don't
really see the reason to block merging it.

**Comment by GaelVaroquaux** [comment]
OK, given that  #2321 has been merged, I'm happy rebasing on master

**Comment by GaelVaroquaux** [comment]
OK. CI is green. Merging

**Comment by GaelVaroquaux** [comment]
@larsoner : this is trying to use your awesome circleci-artifacts-redirector but I'm failing.

If you by chance have time to look at this and / or advise, it would be great (I think that nilearn is in the range of packages that are a bit useful for you). Thanks!!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  circleci_artifacts_redirector_job:
+    runs-on: ubuntu-latest
+    name: Run CircleCI artifacts redirector
+    steps:
+      - name: GitHub Action step
+        uses: larsoner/circleci-artifacts-redirector-action@master
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          artifact-path: 0/doc/index.html
+          circleci-jobs: build_doc
```
Comment (larsoner) [inline .github/workflows/main.yml]: You probably need to use the job names not the step names, so `quick-build,full-build` not `build_doc`

**Pull request merged**